src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -731,4 +731,50 @@ public void testGetDuplicatedItems(final Collection<?> collection, final Set<?>
         final Set<?> result = Utils.getDuplicatedItems(collection);
         Assert.assertEquals(result, duplicated);
     }
+
+    @DataProvider
+    public Object[][] provideDataForTestUtilsSplitString() {
+
+        final String s = "The quick fox jumped over the lazy brown dog.";
+
+        return new Object[][] {
+                { "", "",  "".split("" ) },
+                { "", "1", "".split("1") },
+                { s, "1",   s.split("1") },
+                { s, "",    s.split("" ) },
+                { s, "a",   s.split("a") },
+                { s, "b",   s.split("b") },
+                { s, "c",   s.split("c") },
+                { s, "d",   s.split("d") },
+                { s, "e",   s.split("e") },
+                { s, "f",   s.split("f") },
+                { s, "g",   s.split("g") },
+                { s, "h",   s.split("h") },
+                { s, "i",   s.split("i") },
+                { s, "j",   s.split("j") },
+                { s, "k",   s.split("k") },
+                { s, "l",   s.split("l") },
+                { s, "m",   s.split("m") },
+                { s, "n",   s.split("n") },
+                { s, "o",   s.split("o") },
+                { s, "p",   s.split("p") },
+                { s, "q",   s.split("q") },
+                { s, "r",   s.split("r") },
+                { s, "s",   s.split("s") },
+                { s, "t",   s.split("t") },
+                { s, "u",   s.split("u") },
+                { s, "v",   s.split("v") },
+                { s, "w",   s.split("w") },
+                { s, "x",   s.split("x") },
+                { s, "y",   s.split("y") },
+                { s, "z",   s.split("z") },
+                { s, " ",   s.split(" ") },
```

Comment:
Add test cases for:

-splitting on the final char (`.`)
-splitting on the first char (`T`)
-delimiters of more than 1 char (multiple cases)

Reply:
I've added more text to the test string and added in multiple char delimiters.